docs/guides/high-availability/examples/generated/ispn-site-a.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Source: ispn-helm/templates/infinispan-alerts.yaml
-# tag::fencing-secret[]
+# tag::stonith-secret[]
```

Comment:
This is based on the version of KCB I used (with some additional changes I needed). We can synchronize later. It is kind of a pain to do the synchronization as some things are changed in KCB and not updated here. Maybe we should do some weekly synchronization to no get to state where we need many changes.

Reply:
Should we add the `/stonith/fencing` change to https://github.com/keycloak/keycloak-benchmark/pull/931/files so that this won't be an issue in the future?

Reply:
Ahh, I didn't realize the docs are broken since I changed fencing->stonith. I updated both https://github.com/keycloak/keycloak-benchmark/pull/931/commits/d2e27228ecf7b511f5dcbae969e771703ad6a423